Godeps/_workspace/src/k8s.io/kubernetes/pkg/apiserver/handlers_test.go

```diff
@@ -41,13 +42,21 @@ func (f fakeRL) CanAccept() bool { return bool(f) }
 func (f fakeRL) Accept()         {}
 
 func expectHTTP(url string, code int, t *testing.T) {
-	r, err := http.Get(url)
+	var response *http.Response
+	err := wait.Poll(10, time.Duration(100*time.Millisecond), func() (bool, error) {
+		var e error
+		response, e = http.Get(url)
+		if response.StatusCode == 429 {
```

Comment:
@deads2k I think the failure was caused by 'too many requests' and k8s api refused to serve the request. So I think retrying it should help here. Ideally, our (and upstream) API server returns `Retry-After` header so we can adjust the retry mechanism.